.circleci/scripts/deploy.sh

```diff
@@ -53,4 +61,15 @@ else
   terminus multidev:create $PANTHEON_SITENAME.dev ci-$CIRCLE_BUILD_NUM
   git push -f pantheon $CIRCLE_BRANCH:ci-$CIRCLE_BUILD_NUM
   terminus auth:logout
-fi
\ No newline at end of file
+fi
+
+# copy src back in, remove unnecessary things, commit, tag, and push
+git checkout distribution || git checkout --orphan distribution
+cp -r ../src .
+rm -rf *.txt
+rm -rf components themes *.html
+git add -A
+git commit -m "distribution build: ${GIT_COMMIT_MSG}" --allow-empty
```

Comment:
so a distribution build gets checked in and tagged with every build?  even when working on dev branches?

Reply:
i think it should be just for master, but you are already tagging a version for master.
what the difference between the version in master and distribution build version?

Reply:
master branch has everything (gulp build tasks, etc), distribution only has the dist css js and images and the source scss and template files

Reply:
can they have the same version?
move the code to cleanup build artifacts into the master if clause and just have one version.
unless you foresee a need to have versions for development builds.

Reply:
Yeah now that you mention it I think want to rework this thing.  Thanks for the feedback!